dist/README.md

```diff
@@ -1,6 +1,6 @@
 # `threadsd`
 
-This is the [Threads](https://github.com/textileio/go-threads/tree/master/threadsd) daemon.
+This is the [ThreadsDB](https://github.com/textileio/go-threads/tree/master/threadsd) daemon.
```

Comment:
Are we doing threadsdb or threaddb? I can't remember what we landed on in the end?

Reply:
ThreadsDB with the plural was the consensus